src/components/ec-metroline-card/ec-metroline-card.spec.js

```diff
@@ -0,0 +1,43 @@
+import { mount } from '@vue/test-utils';
+import EcMetrolineCard from './ec-metroline-card.vue';
+
+describe('EcMetrolineCard', () => {
+  function mountMetrolineCard(opts, mountOpts) {
+    return mount(EcMetrolineCard, {
+      propsData: {
+        ...opts,
+      },
+      ...mountOpts,
+    });
+  }
+
+  it('should render as expected', async () => {
+    const wrapper = mountMetrolineCard();
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  it('should render as expected when is collapsed', async () => {
```

Comment:
this is fine, but I prefer to have different describes even if it's just for a new spec "should render properly" nested

Reply:
It makes sense. Updated!